### DISCUSSION TITLE
allow-run-as-system flag official support

### DISCUSSION THREAD

**Discussion by manoharc1999** [MicrosoftEdge/WebView2Feedback#4506] [Q&A] [answered]
Hello
Im planning to use this flag (allow-run-as-system) for one of my application.
When i checked the [documentation](https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/webview-features-flags?tabs=dotnetcsharp) it says it is not recommended to use this flag in production application. 
The flag is very much crucial for my application to work and im afraid it might get removed. 
Can this flag be moved in to official support as it was introduced almost 2 yrs ago?
https://github.com/MicrosoftEdge/WebView2Feedback/issues/1907

**Comment by victorhuangwq** [comment, marked as answer]
Reply as per given in #1907
